x/auctions/types/genesis.go

```diff
@@ -0,0 +1,25 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package types
+
+// DefaultGenesisState sets default auctions genesis state.
+func DefaultGenesisState() *GenesisState {
+	return &GenesisState{
+		Params: Params{
+			EnableAuction: true,
+		},
```

Comment:
add defaults for Bid and round
```suggestion
	return &GenesisState{
		Params: Params{
			EnableAuction: true,
		},
		Bid: Bid{},
		Round: 0,
```